src/ReleaseApp/Infrastructure/Shared/Dto/ReleaseGroupDto.php

```diff
@@ -33,18 +33,30 @@ class ReleaseGroupDto
      */
     protected string $link;
 
+    /**
+     * @var bool
+     */
+    protected bool $conflictDetected = false;
+
     /**
      * @param string $name
      * @param \ReleaseApp\Infrastructure\Shared\Dto\Collection\ModuleDtoCollection $moduleCollection
      * @param bool $containsProjectChanges
      * @param string $link
+     * @param bool $conflictDetected
      */
-    public function __construct(string $name, ModuleDtoCollection $moduleCollection, bool $containsProjectChanges, string $link)
-    {
+    public function __construct(
+        string $name,
+        ModuleDtoCollection $moduleCollection,
+        bool $containsProjectChanges,
+        string $link,
+        bool $conflictDetected = false
```

Comment:
May be rename it to `hasConflict` or smth like that?

Reply:
++